Header header

float64 time
float64 yaw_rate
float64 mdps_torque
float64 mdps_str_ang
float64 VS_CAN
float64 lat_accel
float64 mcp
float64 accel_pedal_value
float64 tps
float64 odometer
float64 battery_voltage
float64 WHL_SPD_RR
float64 WHL_SPD_RL
float64 WHL_SPD_FR
float64 WHL_SPD_FL
